src/functions-and-arrays.js

```diff
@@ -1,42 +1,83 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(x, y) {
+  if(x > y){
```

Comment:
Here, you were not passing the jasmine tests because you were returning more that what was supposed to.
The logic is all there, but you just needed to return the values of the arguments.

```
function maxOfTwoNumbers(x, y) {
  if(x > y){
    return x
  } else {
    return y
  }
}
```